### ISSUE TITLE
Maybe remove the `forked from` subtitle?

### ISSUE THREAD

**Issue by JulienPalard** [Alir3z4/html2text#346]
Hi! Thanks for maintaining html2text!

It can be disturbing to see the "forked from aaronsw/html2text" on the main page, make it looks like unofficial and may cost time to people (round tripping to aaronsw's repo, or to pypi to double check this is the right one).

I recently emailed github support, asking for them to remove it on one of my forks (same pattern: upstream was unmaintained, I'm now the maintainer), and they kindly removed it. So it's possible. if you want it :-)

**Comment by Alir3z4** [comment]
Thank you for mentioning it.

> I recently emailed github support, asking for them to remove it on one of my forks (same pattern: upstream was unmaintained, I'm now the maintainer), and they kindly removed it. So it's possible. if you want it :-)

I didn't know about this, thanks.

One of the reason I didn't want to remove _forked from_ (_or trying it_) or even re-uploading the code to fresh repo in the beginning was due to sentimental reasons I have over Arron story and even though it's mentioned on readme or in git history, I still didn't want to remove his name (_it's not like his name would need such action, he's well known and will always be remembered for many other things he has done_).

[Aaron Swartz died today, 8 years ago](https://news.ycombinator.com/item?id=25731566)

However, 

> It can be disturbing to see the "forked from aaronsw/html2text" on the main page, make it looks like unofficial and may cost time to people (round tripping to aaronsw's repo, or to pypi to double check this is the right one).

Totally makes sense and I never looked at it in that way. Surely it's the right thing.

**Comment by JulienPalard** [comment]
Damn it's a fork from Aaron!? I totally missed the fact it was him, haha, I fully understand your reasons.

So ... I don't know what to say now, I don't want its memories to be erased neither... In the other hand I did *not* recognized it was him by its github handle, so the README with full name may be "enough"...

About the "time lost round-tripping to pypi to ensure this is the right repo", I did it, and that's why I opened the issue in the first place.

**Comment by johnkw** [comment]
I was also confused as to whether this was the correct repo for filing bug reports against the html2text that is delivered in Fedora. It seems it is, even though it looks unofficial.

**Comment by JulienPalard** [comment]
What I think would be great:

- Move « Originally written by Aaron Swartz. This code is distributed under the GPLv3. » on the top of the README,  right before the title.
- Remove the « forked from » subtitle.

Benefits:
- People may know Aaron Swartz from its full name better than from its github handle, at least I do. - It will be clearly displayed on clones and forks too (while the "forked from" is not).
- It would avoid round-trips to pypi to ensure it's a legitimate repo.

**Comment by karolzlot** [comment]
From what I remember when number of stars on this fork will be greater than original repo, then this one will be considered "main" fork in search results (At least it worked this way before). "Forked from" however won't disappear automatically, so this is a way if you want to keep it.

**Comment by mborsetti** [comment]
FWIW I too was totally confused by the "forked" nomenclature of this repo. 

It took me a while to figure out that this was **the** version to use!

**Comment by Alir3z4** [comment]
Thank you all for comments and solutions.

I've asked Github to remove the "forked from" subtitle from the repository.
Later, I'll be applying changes suggested by @JulienPalard in https://github.com/Alir3z4/html2text/issues/346#issuecomment-843264274.

**Comment by eMBee** [comment]
i know this is an old discussion, and the decision is already made and likely can't be undone, but i'd like to provide a counterpoint. when searching for html2text among others i found aarons repo. seeing that it was 14 years old i looked for forks. the only maintained fork that i found there was https://github.com/moregatest/html2text/

of course i also found this repo here, and thought it was an independent implementation. i only realized that they were connected when i saw that both were pointing to the same pypi entry. and then i went through the commit history of that morgatest repo only to find that it was a 100% copy of this very repo here.

in other words, the fact that this repo was not marked as a fork had me utterly confused for a bit, and i had to invest time to untangle that.

if the connection had remained, i would have seen that both  are forks of the same source, and i believe it would also have shown that all the additional commits actually belong to this repo and not the other one.

to put it in other words: removing this repo as a fork caused all the work done here to be miss-attributed to another repo, giving the impression that that repo is very active when in fact it isn't.

**Comment by johnkw** [comment]
That's not a normal way to go looking for an official repo. But if anything there should just be a fork (with lots of "stars" so it's sorted to the top) that says the new official repo has moved to here, and why that happened.

**Comment by eMBee** [comment]
given aarons notoriety i assumed his was the official repo, and i was looking for the most likely successor. i missed the note that this repo is also based on aarons work. perhaps that can be made more prominent as part of the description at the top. (i can make a PR)

another thing that might help (but requires cooperation from someone who can make the change) would be to archive the original repo and include a pointer to here.

this is not very important of course, i am just happy to discover that i get to use one of aarons works, and maybe some day even contribute to it (i am researching some issues i am having)
